common4j/src/main/com/microsoft/identity/common/java/util/IPlatformUtil.java

```diff
@@ -26,12 +26,17 @@
 import com.microsoft.identity.common.java.eststelemetry.LastRequestTelemetryCache;
 import com.microsoft.identity.common.java.exception.ClientException;
 import com.microsoft.identity.common.java.exception.ErrorStrings;
+import com.microsoft.identity.common.java.ui.BrowserDescriptor;
+
+import java.util.List;
 
 import edu.umd.cs.findbugs.annotations.NonNull;
 import edu.umd.cs.findbugs.annotations.Nullable;
 
 public interface IPlatformUtil {
 
+    List<BrowserDescriptor> getBrowserSafeListForBroker();
```

Comment:
```suggestion
    /**
     * Return a list of BrowserDescriptors that are considered safe for the given platform.
     */
    List<BrowserDescriptor> getBrowserSafeListForBroker();
```

Reply:
It should actually be irrelevant for Linux for now. There are no plans to support browser at this time. Only WebView for now.

Reply:
Sure.  It can be empty for linux, but the alternative is to distinguish the actual parameters and builder per-platform, I think.  I'm not sure that's what we want.

Reply:
I think for Linux it's empty.  Otherwise we need to talk about whether it makes sense to have the same parameters objects for each platform.  Maybe it does not, but that's not the path we are on.